src/streams/SubscriptionsScreen.js

```diff
@@ -28,14 +33,47 @@ const styles = createStyleSheet({
     flex: 1,
     flexDirection: 'column',
   },
+  rightItem: {
+    marginLeft: 'auto',
+  },
+  rightIcon: {
+    marginLeft: 'auto',
+    color: '#666699',
+  },
+  allStreamsButton: {
+    paddingRight: 12,
+  },
+  streamsText: {
+    textTransform: 'uppercase',
+    color: '#666699',
+    fontWeight: '500',
+  },
 });
 
 type Props = $ReadOnly<{|
-  navigation: StreamTabsNavigationProp<'subscribed'>,
+  navigation: AppNavigationProp<'subscribed'>,
   route: RouteProp<'subscribed', void>,
 |}>;
 
-export default function SubscriptionsCard(props: Props): Node {
+type FooterProps = $ReadOnly<{||}>;
+
+function AllStreamsButton(props: FooterProps): Node {
+  const navigation = useNavigation();
+  const handlePressAllScreens = useCallback(() => {
+    navigation.push('all-streams');
+  }, [navigation]);
+
+  return (
+    <Touchable onPress={handlePressAllScreens}>
+      <View style={[appStyles.listItem, styles.allStreamsButton]}>
+        <ZulipTextIntl style={styles.streamsText} text="All Streams" />
```

Comment:
nit: Let's make the text "All streams". That's our usual style for UI text; here it doesn't affect what's displayed, because of the all-caps transform, but it does make for a small bit less work for our translation contributors because we already have the string "All streams".